GitUI/Script/SplitButton.cs

```diff
@@ -321,7 +321,7 @@ protected override void OnPaint(PaintEventArgs pevent)
                               bounds.Width - _dropDownRectangle.Width - internalBorder,
                               bounds.Height - (internalBorder * 2) + 2);
 
-            bool drawSplitLine = State == PushButtonState.Hot || State == PushButtonState.Pressed || !Application.RenderWithVisualStyles;
+            bool drawSplitLine = State is PushButtonState.Hot or PushButtonState.Pressed || !Application.RenderWithVisualStyles;
```

Comment:
```suggestion
            bool drawSplitLine = !Application.RenderWithVisualStyles || (State is PushButtonState.Hot or PushButtonState.Pressed);
```

Reply:
Intensely agree with the suggestion here. Parentheses are essential.

Reply:
Added parenthesis in general. Did not change the order here (kept as below)

```C#
bool drawSplitLine = State is(PushButtonState.Hot or PushButtonState.Pressed) || !Application.RenderWithVisualStyles;
```